### DISCUSSION TITLE
Two questions re the UI

### DISCUSSION THREAD

**Discussion by michaelbeijer** [xiaoyifang/goldendict-ng#446] [Q&A] [answered]
**(1.)** Is there any way to navigate up/down through the list of terms in the Search Pane via the keyboard?

![up and down](https://user-images.githubusercontent.com/16622240/229379538-ba83e79a-05e4-4eb3-b8e4-8b55ed65477c.png)

**(2.)** Is there a way to keep all dictionary tabs shown? That is, so instead of having to click the little arrw to reveal more, they are all shown, stacked in menus bars?

![dictionary-names-stacked](https://user-images.githubusercontent.com/16622240/229379462-14e0d8ec-188e-4377-8853-58b0fa238535.png)

Michael

**Comment by shenlebantongying** [comment, marked as answer]
1.) Fixed.

2.) It seems other users use "Groups" to mass enable/disable dictionaries (Some people uses like 100+ dictionaries). If the dictionaries are too much, it is not manageable by clicking them one by one anyway.

**Reply by michaelbeijer** [reply]
Thanks for answering, shenlebantongying!

Regarding #1, I just realised that I can sort of do it by pressing the Esc key, and then immediately pressing UP/DOWN!

Regarding #2, I agree, it would quickly become unmanageable. Hadn't thought of that. And since I plan to use several hundreds of dictionaries myself, I think I will look into the Groups feature. 

Incidentally, for an idea of how many resources I have, have a look at my terminology wiki: https://termhotel.com/Home

Michael
